pennylane/pauli/dla/structure_constants.py

```diff
@@ -187,178 +177,10 @@ def structure_constants(
-            computational cost. Defaults to ``True``.
-
-    Returns:
-        TensorLike: The adjoint representation of shape ``(d, d, d)``, corresponding to
-        the indices ``(gamma, alpha, beta)``.
-
-    **Example**
-
-    Let us generate the DLA of the transverse field Ising model using :func:`~lie_closure`.
-
-    >>> n = 4
-    >>> gens = [qml.X(i) @ qml.X(i+1) + qml.Y(i) @ qml.Y(i+1) + qml.Z(i) @ qml.Z(i+1) for i in range(n-1)]
-    >>> g = qml.lie_closure(gens, matrix=True)
-    >>> g.shape
-    (12, 16, 16)
-
-    The DLA is represented by a collection of twelve :math:`2^4 \times 2^4` matrices.
-    Hence, the dimension of the DLA is :math:`d = 12` and the structure constants have shape ``(12, 12, 12)``.
-
-    >>> adj = qml.structure_constants(g, matrix=True)
-    >>> adj.shape
-    (12, 12, 12)
-
-    **Internal representation**
-
-    As mentioned above, the input is assumed to be a batch of Hermitian matrices, even though
-    algebra elements are usually skew-Hermitian. That is, the input should represent the operators
-    :math:`G_\alpha` for an algebra basis :math:`\{iG_\alpha\}_\alpha`.
-    In an orthonormal basis of this form, the structure constants can then be computed simply via
-
-    .. math::
-
-        f^\gamma_{\alpha, \beta} = \text{tr}[-i G_\gamma[iG_\alpha, iG_\beta]] = i\text{tr}[G_\gamma [G_\alpha, G_\beta]] \in \mathbb{R}.
-
-    Possible deviations of an orthogonal basis from normalization is taken into account in a
-    reduced version of the step for non-orthogonal bases below.
-
-    **Structure constants in non-orthogonal bases**
-
-    Structure constants are often discussed using an orthogonal basis of the algebra.
-    This function can deal with non-orthogonal bases as well. For this, the Gram
-    matrix :math:`g` between the basis elements is taken into account when computing the overlap
-    of a commutator :math:`[iG_\alpha, iG_\beta]` with all algebra elements :math:`iG_\gamma`.
-    The resulting formula reads
-
-    .. math::
-
-        f^\gamma_{\alpha, \beta} &= \sum_\eta g^{-1}_{\gamma\eta} i \text{tr}[G_\eta [G_\alpha, G_\beta]]\\
-        g_{\gamma \eta} &= \text{tr}[G_\gamma G_\eta] \quad(\in\mathbb{R})
-
-    Internally, the commutators are computed by evaluating all operator products and subtracting
-    suitable pairs of products from each other. These products can be reused to evaluate the
-    Gram matrix as well.
-    For orthogonal but not normalized bases, a reduced version of this step is used, only
-    computing (and inverting) the diagonal of the Gram matrix.
-    """
+    warnings.warn(
+        "Calling structure_constants via ``qml.pauli.structure_constants`` is deprecated. ``structure_constants`` has moved to ``pennylane.liealg``. "
+        "Please call ``structure_constants`` from top level as ``qml.structure_constants`` or from the liealg module via ``qml.liealg.structure_constants``.",
+        qml.PennyLaneDeprecationWarning,
```

Comment:
Are the backticks (``) useful here? They do highlight that this is code, but this comes up in the command line or jupyter notebook output so no formatting is applied, correct?

Reply:
Yeah good point 🤔 will remove!